threadpoolctl.py

```diff
@@ -660,6 +667,19 @@ def _make_controller_from_path(self, filepath):
             if prefix is None:
                 continue
 
+            # workaround for blas install from conda-forge on windows. They rename all
+            # blas libs into "libblas.dll". We then have to check to which blas
+            # implementation it corresponds.
+            if prefix == "libblas":
+                if filename.endswith(".dll"):
```

Comment:
I wonder if we should stay windows-specific or try to be blindly future-proof and do this for any platform. I think I would favor the later which should also be simpler from a code point of view. WDYT?

Reply:
I had to make it windows specific because there was a "libblas.so.3" on linux that came from "/usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3" that does not expose all the symbols we want. I don't know what it is exactly. See https://dev.azure.com/joblib/threadpoolctl/_build/results?buildId=1395&view=logs&j=49602bd8-c265-5cf5-ff6d-9ba0d4e0f8a9&t=7b75b0ca-af55-5daa-918c-61fa8f39fc20

Reply:
Maybe add an inline comment for this.

Reply:
Why point was more to mention that ubuntu (and maybe debian) also includes a `libblas` as part of its openblas distribution with some but not all the openblas specific symbols in addition to a `libopenblas` that comes with all symbols.